R/cube-transforms.R

```diff
@@ -86,37 +118,187 @@ setMethod("subtotalArray", "CrunchCube", function(x, headings = FALSE) {
 #'
 #' @aliases subtotalArray
 #' @export
-applyTransforms <- function (x, array = cubeToArray(x), ...) {
-    # if any of the dimensions are subvariables, don't even attempt to calculate
-    # transforms or insertions.
-    if (any(getDimTypes(x) == "ca_items")) {
-        return(array)
-    }
+applyTransforms <- function (x,
```

Comment:
I've noticed several places where you've replaced `x <- as.array(cube)` with `x <- applyTransforms(x)`. Should `as.array.CrunchCube <- applyTransforms`?

Reply:
I started down that path in this PR, but found there were a bunch of little things I would have to change. I think that's probably a good idea, but I think we should defer it if possible and do it separately for ease of review.